pyccel/ast/builtin_methods/list_methods.py

```diff
@@ -290,6 +300,16 @@ def __init__(self, list_obj) -> None:
         self._class_type = list_obj.class_type
         super().__init__(list_obj)
 
+    @property
+    def modified_args(self):
+        """
+        Return an iterable of all the arguments which may be modified by this function.
+
+        Return an iterable of all the arguments which may be modified by this function.
+        This is notably useful in order to determine the constness of arguments.
+        """
+        return ()
```

Comment:
I may have misunderstood the meaning of `modified_args`.

When we append an object to a list, the list itself is modified. So why do we not return `(self._list_obj,)` here?

Reply:
By the way
```suggestion
    @property
    def modified_args(self):
        """
        Return a tuple of all the arguments which may be modified by this function.

        Return a tuple of all the arguments which may be modified by this function.
        This is notably useful in order to determine the constness of arguments.
        """
        return ()
```

Reply:
This is not a method of `ListAppend`. It is a method of `ListCopy`. When we copy a list the list itself is not modified

Reply:
🤦‍♂️